src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs

```diff
@@ -29,45 +29,45 @@ namespace Microsoft.ApplicationInsights.Extensibility.Implementation.External
     using System.Collections.Concurrent;
     using System.Collections.Generic;
 
-    
-    
+
```

Comment:
I would undo all changes in this file. Even though there are only whitespace changes, it's still different from what was auto-generated by the tool and could cause problems.